index.d.ts

```diff
@@ -457,7 +468,7 @@ declare module 'plaid' {
   }
 
   interface ClientOptions extends CoreOptions {
-    version?: '2018-05-22' | '2017-03-08';
+    version?: '2019-05-29';
```

Comment:
I think we need to change the default version somewhere else in the client too? this is only the typescript typings, not the actual implementation.

Reply:
i can't see it set anywhere; it looks like you have to explicitly set it in `options` or it uses the default on theserver.

Reply:
Looking at this: https://plaid.com/docs/api-upgrades/#versioning
It doesn't sound like the node client is supposed to have a default, it has to be initialized with a specific version

Reply:
Interesting. does that makes sense?

I know the api will default to using the client version they have configured on the dashboard. But consider the case where a client has an older version configured, and then upgrades their node version. They'd expect to get API responses conforming with the new typescript typings, but this wouldn't be the case until they supplied the version explicitly, which could be difficult to debug.

I think we should add `2019-05-29` it as the default option if its not supplied. I think this is the behavior of the other clients... Thoughts @tanatplaid @jasonjia93 ?

Reply:
:zap: we've added a default option which should make sense with a version upgrade

Reply:
👍